stdlib/asm/math/ext2.masm

```diff
@@ -0,0 +1,84 @@
+# Given a stack with initial configuration given by [a1,a0,b1,b0,...] where a = (a0,a1) and
+# b = (b0,b1) represent elements in the extension field of degree 2, the procedure outputs the 
+# product c = (c1,c0) where c0 = a0b0 - 2(a1b1) and c1 = (a0 + a1)(b0 + b1) - a0b0
+export.mul_ext_2
```

Comment:
Since the module is already called `ext2` we probably don't need the `ext_2` suffix. Otherwise, the exec instruction would like like `exec.ext2::mul_ext_2`.

Reply:
makes sense, thank you.